icu4j/main/core/src/main/java/com/ibm/icu/impl/Grego.java

```diff
@@ -180,6 +189,10 @@ public static int[] timeToFields(long time, int[] fields) {
         return fields;
     }
 
+    public static Pair<Integer, Integer> timeToYear(long time) {
+        return dayToYear(floorDivide(time, 24*60*60*1000 /* milliseconds per day */, new long[1]));
```

Comment:
There's nowhere where you're calling this function and doing anything with the second element of the pair.  Is it worth having it just return the first element in the pair returned by `dayToYear()`, rather than doing that at all the call sites?